### ISSUE TITLE
Asset party model / material details tab.

### ISSUE THREAD

**Issue by NotGaylien** [Facepunch/sbox-issues#3723]
### For?

Asset Party

### What can't you do?

Currently we dont have that much information on asset party regarding models and materials on the website.

we have a few bits, but when looking at them there is alot we dont know about the model till we download it.
material groups, bodygroups, animations, animgraph, material count, attachment points and so on.

Animgraph and material details are a massive one we have no idea really about and they should ideally be able to be known to people.

### How would you like it to work?

Have a new Details tab on models and materials and so on,
which contains all the useful information that may or may not be useful to every single person.
but is still extremely useful for those who might need it.
Or are just interested.

![image](https://github.com/sboxgame/issues/assets/82092361/24a663d8-e241-41b1-a8d8-f050bdea53d6)

Like this but obviously a whole lot clearer and better.
i am not good at designing ui....

### What have you tried?

We can always download the models in editor and view a decent chunk of that stuff,
but then i have to download it first to see that, and i need to have the editor open.
it would be nice to be able to go around see good models and favourite them on the website to use later on.

### Additional context

Related to this https://github.com/sboxgame/issues/issues/3721
basically same thing as that, but on the website as well.
